syntax = "proto3";

option java_package = "proto";
message PricesStreamedFileHeader {
    string request_id = 1;
    string timestamp = 2;
    string source = 3;
    
}